gpt_synthesizer/main.py

```diff
@@ -89,6 +147,7 @@ def main(verbose = False):
 
         spec_memory.return_messages = True
         sum_output = summary_chain.predict(input=spec_memory.load_memory_variables({}))
+        all_summary += sum_output
```

Comment:
It is not clear what is the role of 'all_summary'. I guess it was supposed to be fed to the main_generator.

Reply:
That's right! My mistake.